### ISSUE TITLE
EFC Sim: Element selection animations sometimes go wacky after a reset in Energy Systems tab

### ISSUE THREAD

**Issue by pwolfert** [Connexions/simulations#45]
This has been a really difficult bug to track down.  I'm still not sure where it comes from.  At some point after resetting, the element "carousels" can start moving sideways or diagonally.  Basically, they're only supposed to have translation in the y-axis, but some x-axis movement is getting mixed in there.

**Comment by pwolfert** [comment]
And every time I set things up to catch it in the act, I can't reproduce the error!

**Comment by pwolfert** [comment]
The position-shifting problem:
![screen shot 2015-01-20 at 11 00 30 am](https://cloud.githubusercontent.com/assets/7595652/5818135/b20c5060-a0aa-11e4-84d0-bf16b1c52ee8.png)

The energy-chunk-insanity problem:
![screen shot 2015-01-20 at 1 43 49 pm](https://cloud.githubusercontent.com/assets/7595652/5818134/b1fb8a64-a0aa-11e4-90a6-147eb8ce18f8.png)

**Comment by pwolfert** [comment]
Just for the sake of records, these are before-and-after positions of the generator and solar panel after it freaks out.  The y positions are so arbitrary that it makes me think the vectors themselves are being defiled somehow.

```
electrical-generator: (-0.0250, 0.0000)
electrical-generator: (-0.0250, -0.0178)
solar-panel: (-0.0250, -0.4000)
solar-panel: (-0.0250, 0.0648)
```

Also, I was investigating whether it was indeed the CarouselAnimator's animation or if it was something else, and so I disabled the animating part and made it apply the end transformations instantly, and I found that it still messed everything up when applying them instantly.
